packages/styles/components/_c.checkbox.scss

```diff
@@ -12,6 +12,11 @@
     margin-left: magic-unit-rem(0.5, true);
   }
 
+  &__suffix-label {
```

Comment:
As the html element is already inside the label tag, we can just name this class `__suffix`.